runtime/Cpp/runtime/src/atn/SemanticContext.h

```diff
@@ -144,6 +144,8 @@ namespace atn {
    */
   class ANTLR4CPP_PUBLIC SemanticContext::Operator : public SemanticContext {
   public:
+    ~Operator();
```

Comment:
Good reasoning. I prefer explicit code (and use that in all my projects). So for a base virtual method I use `virtual method();` and for overrides `virtual method() override;`. But for virtual d-tors I often missed to use **override**, which is a mistake. Have to take more care for that.

For the antlr runtime I'd like to use the same approach, so this (and similar places) should be changed to `virtual ~Operator() override;` even though `virtual` seems to be superfluous here (but aligns nicely visually with other virtuals).